bears-router.js

```diff
@@ -0,0 +1,84 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: './data/lambda.sqlite3',
+    },
+    // debug: true,
+};
+const db = knex(knexConfig);
+
+router.post('/', (req,res) => {
+    if (!req.body.name) {
+        res.status(400).json({error:'missing required field: name'});
+    } else {
+    db('bears').insert(req.body)
```

Comment:
it works fine referencing the zero-index of the ids that come back, but you could also tack on the `.first()` method at the end of this db call like you do for the get by Id a couple lines further down